index.html

```diff
@@ -8,29 +8,33 @@
     <title>scotty.im</title>
   </head>
   <body>
-    <header>
-      <div class="logo"></div>
-      <div class="slogan"></div>
-    </header>
-    <main>
-      <div>
-        <h2>Hi, I am Scotty.</h2>
-        <h3 class="device__desktop">
-          Welcome to my teleportation service for Hybrid Multi-Cloud Management!
-        </h3>
-        <h3 class="device__desktop">
-          I can deliver your cloud-native apps anywhere you want.
-        </h3>
-        <h3 class="device__mobile">
-          Welcome to my teleportation service <br />
-          for Hybrid Multi-Cloud Management!
-        </h3>
-        <h3 class="device__mobile">
-          I can deliver your cloud-native apps <br />
-          anywhere you want.
-        </h3>
-      </div>
-      <p>Coming Soon.</p>
-    </main>
+    <div class="background">
+      <header>
+        <div class="logo"></div>
+        <div class="slogan"></div>
```

Comment:
이런 이미지로 사용하는 엘리먼트들에 대해서는 
```suggestion
        <div class="slogan" role="img" aria-label="slogan"></div>
```
와 같이 ARIA attribute들을 추가해주는게 좋습니다

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Role_Img